### PR TITLE
Updated Datek HSE2927E and added support for Datek Window/Door sensor HSE2920E

### PR DESCRIPTION
**Pull request by TheStigh**
Added support for Led-On-Motion for Datek Motion Sensor HSE2927E

### CODE REVIEW

**Comment by TheStigh** [comment]
Please merge #5216 first.

- Added support for Led-On-Motion for Datek Motion Sensor HSE2927E
- Updated Datek Motion Sensor (HSE2927E) and added support for  Datek Window Sensor (HSE2920E)

File: devices/datek.js [hunk 1]
```diff
@@ -85,13 +85,32 @@ module.exports = [
             e.voltage_phase_c(), e.temperature()],
     },
     {
-        fingerprint: [{modelID: 'Motion Sensor', manufacturerName: 'Eva'}],
+        zigbeeModel: ['Motion Sensor'],
```
Comment (Koenkk) [inline devices/datek.js:88]: Was this removed on purpose? (preferable this should be kept to prevent conflicts with devices using the same modelID)

Comment (TheStigh) [reply devices/datek.js:88]: Yes it was. No reason to actually remove it, do you want me to put it back ?

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**